packages/react-components/src/components/RemoteVideoTile.tsx

```diff
@@ -52,12 +57,13 @@ export const _RemoteVideoTile = React.memo(
       userId,
       displayName,
       onRenderAvatar,
-      showMuteIndicator,
-      /* @conditional-compile-remove(one-to-n-calling) */
-      /* @conditional-compile-remove(PSTN-calls) */
-      participantState
+      showMuteIndicator
     } = props;
 
+    const containerRef = React.useRef<HTMLDivElement>(null);
+    /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(PSTN-calls) */
+    const containerWidth = _useContainerWidth(containerRef);
```

Comment:
This gets the container width, but also requires an `isNarrow` prop be passed in - can just one of these be required?

Reply:
(is isNarrow necessary or can we just use the container width here?)

Reply:
(better would be if the parent could just use the showLabel prop but I'm guessing the parent won't know this tile's width well enough)

Reply:
Good idea @JamesBurnside 
isNarrow does seem redundant now. 
Will remove it.
And yes, doing this inside video gallery requires quite a bit of refactor.